docs/guides/workflows_patterns/deployment_airflow.rst

```diff
@@ -1,17 +1,106 @@
 .. _deloyment_airflow:
 
-.. warning:: This doc is a stub.
-
-#################################
 Deploying Great Expectations with Airflow
-#################################
+=========================================
+
+This guide will help you deploy Great Expectations within an Airflow pipeline. You can use Great Expectations to automate validation of data integrity and navigate your DAG based on the output of validations.
+
+.. admonition:: Prerequisites: This workflow pattern assumes you have already:
+
+    - Configured a :ref:`Data Context <tutorials__getting_started__initialize_a_data_context>`.
+    - Configured an :ref:`Expectations Suite <tutorials__getting_started__create_your_first_expectations>`.
+    - Optional: Configured a :ref:`Checkpoint <tutorials__getting_started__set_up_your_first_checkpoint>`.
+    - Configured an Airflow pipeline (DAG).
+
+There are two supported methods: using an Airflow ``PythonOperator`` to run Validations using python code or invoking the Great Expectations CLI to run a Checkpoint using an Airflow ``BashOperator``.
+
+.. note::
+    There is not currently a Great Expectations custom Airflow operator, however we have heard from users who have implemented their own. We love community contributions so please share your operators!
+
+Check out these links for some examples of Airflow pipelines with Great Expectations Validations ensuring that downstream tasks are protected from upstream issues:
+
+- `Great Expectations Pipeline Tutorial <https://github.com/superconductive/ge_tutorials>`_
```

Comment:
Question for @jcampbell, should we be referencing both or would one of them be sufficient/our source of truth going forward?

Reply:
I'm interested to hear James' input as well. Just for context, I included the one from the meetup because it had a sweet parameterized validate method: https://github.com/superconductive/airflow_meetup_demo/blob/master/demo/dags/dag_definition_2.py#L22
But I'm happy to remove the second tutorial and: add that validate function to this doc, to the pipeline tutorial or just skip it.

Reply:
I'd like to remove the second one and focus only on newer tutorial. In fact, it may be time to just sunset the airflow meetup demo repo.